pulsar/consumer_partition.go

```diff
@@ -1113,6 +1114,17 @@ func (pc *partitionConsumer) discardCorruptedMessage(msgID *pb.MessageIdData,
 		})
 }
 
+func (pc *partitionConsumer) _setConn(conn internal.Connection) {
+	pc.conn.Store(conn)
+}
+
+func (pc *partitionConsumer) _getConn() internal.Connection {
```

Comment:
Maybe we don’t need _ as a function prefix

Reply:
@jonyhy96 ,

I wanted to make it clear that these methods should not be used by other components in the `pulsar` package, but I could remove this if you don't like it.  Admittedly this style of naming is not used anywhere else in the codebase.

Reply:
> I wanted to make it clear that these methods should not be used by other components in the pulsar package
Agree with this, but this is not a guarantee. Maybe we can add some comment on this function.

Reply:
Sure; comments added.